_jobs/positions/__QA-testing-engineer.md

```diff
@@ -0,0 +1,54 @@
+---
+title: QA Testing & Automation Engineer
+is_fellow: true
+lever_link:
+position: 1
+lede_markdown: We’re looking for skilled manager and mentor to lead engineering practices for City of Austin’s Design, Technology, and Innovation Fellows program.
```

Comment:
```suggestion
lede_markdown: We’re looking for a skilled manager and mentor to lead engineering practices for City of Austin’s Office of Design and Delivery.
```

Reply:
Oh this description is leftover copy from the engineering lead position. :-D

Reply:
@srigdon do you have a suggestion for summary copy of this position?

Reply:
What about: 

We're looking for a skilled quality assurance engineer to collaborate with the City of Austin’s Office of Design and Delivery on incorporating effective testing practices into every development cycle. If you're interested in delivering services that meet and adapt with residents' needs, read on and apply!